packages/rulesets/generated/spectral/az-dataplane.js

```diff
@@ -2201,6 +2215,8 @@ const ruleset = {
             description: "All schema properties should have a defined type.",
             message: "Property should have a defined type.",
             severity: "warn",
+            disableForTypeSpec: true,
+            disableForTypeSpecReason: "TODO",
```

Comment:
Not exactly sure what this rule check but this might just be covered by typespec syntax that requires you to give a type to a property.